Cargo.toml

```
@@ -11,7 +11,7 @@ keywords = ["paillier", "homomorphic", "encryption", "zero-knowledge", "cryptoag
 [dependencies]
 rayon = "1.1"
 serde = { version = "1.0", features = ["derive"] }
-curv = { package = "curv-kzen", version = "0.7", default-features = false }
+curv = { package = "curv-kzen", version = "0.8", default-features = false }
```

Since curv@0.8 specifying `package = "curv-kzen" ` is not necessary. We can shorter this line:
```suggestion
curv-kzen = { version = "0.8", default-features = false }
```

No other changes should be required

---

Needed to tweak the feature reference to match too :+1: